src/main/java/frc/robot/subsystems/arm/commands/JoystickControl.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class JoystickControl extends CommandBase {
+    private final Arm arm;
+    private final Joystick joystick;
+
+    public JoystickControl(Arm arm, Joystick joystick) {
+        this.arm = arm;
+        this.joystick = joystick;
+    }
+
+    @Override
+    public void execute() {
+        arm.setShoulderJointPower(joystick.getY());
+        arm.setElbowJointPower(joystick.getY());

```

Comment:
Remember, Don't Repeat Yourself. `joystick.getY()` should be stored in a variable.
However, the best way is to have a "throttle" supplier as a parameter of the command, then get its value here. The `XboxControl` command is pretty much identical to this command, a supplier can be used for Xbox input too.